_posts/newsletters/2024-02-29-newsletter.md

```diff
@@ -0,0 +1,519 @@
+**The February Community Call was on the topic of Keeping Up With Changing Technology.**
+
+Speakers presented on experiences where they had to make the call on when to upgrade a technology, change a framework, or let a project end.
+
+
+<a name="community-manager"></a>
+## Community Manager Office Hours
+One weekday and one weekend time slot will be set aside each month for US-RSE members to chat with Sophia about membership matters. Members may also provide updates on how they are applying technical tools and other resources they have learned about from the US-RSE community. Additional information and relevant resources can be found on the US-RSE Slack in the [#community-manager](https://app.slack.com/client/T8ZT4PJSW/C06E5P599JN) channel.
+
+<a name="rse-gln"></a>
+## RSE Group Leaders' Network (RSE-GLN)
+Do you lead an RSE group?  Do you want to talk to others about the challenges of leading and managing projects, people, and resources?  [Join the RSE-GLN](https://us-rse.org/ag/rse-gln/)!
+
+<a name="funder-series"></a>
+## US-RSE Funders Talk Series
+The US-RSE hosted a series of conversations with funders for 1) RSEs to learn about funding opportunities and review and other processes, 2) RSEs to ask questions of the funder, and 3) funders to ask questions and learn about RSEs.
+
+Visit the [Funder Talk Series](https://us-rse.org/wg/outreach/#funder-series) for more information and links to past presentations. 
+
+If you have suggestions for future talks, please contact [Dan Katz](mailto:d.katz@ieee.org).
+
+*This series is now on hiatus through the end of 2024. We'll start thinking about possible next steps & future talks for 2025 later this year.*
+
+<a name="events"></a>
+# **Interesting Events and Opportunities**
+
+## Community and Travel Funds are available
+
+Part of the Alfred P. Sloan Foundation grant for US-RSE has been delegated for the [Community and Travel Funds program](https://us-rse.org/grants-and-awards/). Members of US-RSE can apply for funds for community or individual purposes for event costs, get-togethers, travel funding, and more. Apply today!
+
+## Nominations are open for the US-RSE Community Awards program
+In 2024, US-RSE is proud to unveil a new initiative aimed at honoring individuals who have significantly contributed to the Research Software Engineering community. This initiative is designed to recognize their dedication and support in furthering the mission and objectives of US-RSE.
+
+The US-RSE Community Awards are intended for individuals actively advancing the mission of US-RSE. There are two awards available. The Excellence in Service Award acknowledges outstanding dedication to advancing the US-RSE mission. The Impact Award recognizes notable contributions and influence within the RSE profession. 
+
+Nominations for these prestigious awards are open until March 15th. To nominate someone, including yourself, please visit the event website: https://us-rse.org/community-awards/.
```

Comment:
Suggest deleting these. They show up earlier.

Reply:
Thank you. I have removed them.